sync.js

```diff
@@ -62,6 +62,8 @@ GlobSync.prototype._finish = function () {
         } catch (er) {
           if (er.syscall === 'stat')
             set[self._makeAbs(p)] = true
+          else if (er.code === 'ELOOP')
+            if (real) set[real] = true
           else
```

Comment:
What is the intent of this bit?  The indentation is incorrect, making it look like any error that isn't a `stat` or `ELOOP` will get thrown, but in fact, the only error that'll get thrown is an `ELOOP` when `real` is falsey.